stack/rest/src/test/java/org/apache/usergrid/rest/test/resource2point0/endpoints/NamedResource.java

```diff
@@ -248,6 +250,29 @@ public ApiResponse  post(boolean useToken, Map map, QueryParameters queryParamet
 
     }
 
+    //Used for empty posts
+    public <T> T post( boolean useToken, Class<T> type, Map entity, final QueryParameters queryParameters, boolean useBasicAuthentication ) {
```

Comment:
Overloaded the existing method to save effort on refactoring it all. Should go away when we tear the tokens out of the post calls as authorization will be handled seperately.